### PR TITLE
Query-Node: members_LeaderInvitationQuotaUpdated implementation

### PR DESCRIPTION
**Pull request by mnaamani**
Complete mappings implementation of `members_LeaderInvitationQuotaUpdated` to update quota. This fixes query-node checks failing for the faucet init flow on playground:

https://github.com/Joystream/joystream/runs/5412009311?check_suite_focus=true#step:9:1075

You can see the negative value indicates the inviteCount was not update before the transfer event occurred.

```
2022-03-03T18:50:00.814Z integration-tests:job:Initialize Faucet flow 0 failed:

AssertionError: expected -95 to equal 10

at /home/runner/work/joystream/joystream/tests/network-tests/src/fixtures/membership/TransferInvitesHappyCaseFixture.ts:78:16
```

### CODE REVIEW

**Comment by mnaamani** [comment]
I can't make sense of why integration tests are failing in this way resulting from this change:

```
Job Results:
electing council: Succeeded
membership system: Succeeded
creating members: Succeeded
updating member profile: Succeeded
updating member accounts: Succeeded
inviting members: Succeeded
transferring invites: Succeeded
managing staking accounts: Succeeded
electing second council: Succeeded
council election failures: Succeeded
proposals & proposal discussion: Failed
sudo lead opening: Failed
openings and applications: Skipped
upcoming openings: Skipped
group status: Skipped
worker actions: Skipped
group budget: Skipped
updating member verification status: Failed
forum categories: Skipped
forum threads: Skipped
forum thread tags: Skipped
forum polls: Skipped
forum posts: Skipped
forum moderation: Skipped
check active video counters: Skipped
nft auction and offers: Failed // understandable because we have disabled nft functionality
create channel via CLI: Failed
init storage and distribution buckets via CLI: Succeeded
Error: Scenario Failed
```

File: query-node/mappings/src/membership.ts [hunk 1]
```diff
@@ -472,6 +473,18 @@ export async function members_LeaderInvitationQuotaUpdated({
 }: EventContext & StoreContext): Promise<void> {
   const [newQuota] = new Members.LeaderInvitationQuotaUpdatedEvent(event).params
 
+  const groupName = 'membershipWorkingGroup'
+  const group = await store.get(WorkingGroup, { where: { name: groupName }, relations: [] })
```
Comment (Lezek123) [inline query-node/mappings/src/membership.ts]:
```suggestion
  const group = await store.get(WorkingGroup, { where: { name: groupName }, relations: ['leader', 'leader.membership'] })
```

**Review by Lezek123 — CHANGES_REQUESTED** [review]
Thank you for this, I must've somehow missed that part

**Comment by Lezek123** [comment]
>  can't make sense of why integration tests are failing in this way resulting from this change:

Probably processor dying due to error mentioned above

**Pull request merged**